.travis.yml

```diff
@@ -80,6 +77,3 @@ script:
 
 after_success:
   - if [[ $CODECOVERAGE == 1 ]]; then bash <(curl -s https://codecov.io/bash); fi
-
-notifications:
-  email: false
```

Comment:
This seems to be mistakenly removed.

Reply:
Nope, that is void stuff that is not needed :) Trust me.

Reply:
I'll take your word for it, just want to start getting notification emails for every Travis build :)

Reply:
I feel you : )

Reply:
I will double check to make sure.